src/EventBuilder/Platforms/WPF.cs

```diff
@@ -7,20 +7,21 @@
 
 namespace EventBuilder.Platforms
 {
-    public class Net45 : BasePlatform
+    public class WPF : BasePlatform
     {
-        public Net45()
+        public WPF()
         {
             if (PlatformHelper.IsRunningOnMono())
             {
-                throw new NotSupportedException("Building events for NET45 on Mac is not implemented yet.");
+                throw new NotSupportedException("Building events for WPF on Mac is not implemented yet.");
             }
             else
             {
                 Assemblies.Add(@"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\WindowsBase.dll");
                 Assemblies.Add(@"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\PresentationCore.dll");
                 Assemblies.Add(@"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\PresentationFramework.dll");
 
+
```

Comment:
Mostly because I have to find something....extra new line?

Reply:
Events_WinForms.cs is missing from ReactiveUI.Events_WinForms project.